tests/upgrade_test.go

```diff
@@ -7,16 +7,18 @@ import (
 
 	"github.com/canonical/matter-snap-testing/env"
 	"github.com/canonical/matter-snap-testing/utils"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestUpgrade(t *testing.T) {
 	start := time.Now()
 
 	t.Cleanup(func() {
+		// Remove snaps, ignoring errors
 		utils.SnapRemove(nil, allClustersSnap)
-		utils.SnapDumpLogs(nil, start, allClustersSnap)
-
 		utils.SnapRemove(nil, chipToolSnap)
+
+		utils.SnapDumpLogs(t, start, allClustersSnap)
```

Comment:
Ignoring only chip-tool errors?

Reply:
Maybe the comment is unclear. The goal is to allow the SnapRemove to continue even if it fails, but the SnapDumpLogs may cause a test fail.

The comment above the two remove lines could perhaps rather read:
```
// Remove snaps, ignoring errors during removal
```